docs/release-notes/8.11.asciidoc

```diff
@@ -1,6 +1,23 @@
 [[release-notes-header-8.11.0]]
 == 8.11
 
+[discrete]
+[[release-notes-8.11.2]]
+=== 8.11.2
+
+[discrete]
+[[enhancements-8.11.2]]
+==== Enhancements
+* Correct References under 'Useful Links' in Risk Score Enablement page ({pull}171089[#171089]).
```

Comment:
@tiansivive and @SourinPaul can you share a more descriptive summary for this change? I just need a one-liner describing what was corrected or improved with https://github.com/elastic/kibana/pull/171089.

Reply:
@nastasha-solomon  We've changed the links to point to the correct in-app pages instead, and removed the distinction between host/user, replacing it with Entity.

Reply:
@SourinPaul @tiansivive is this OK? 
> Updates in-app reference links on the Entity scoring enablement page.

Reply:
Thanks, @tiansivive! Making the following changes:

```suggestion
* Updates documentation references on the Entity risk score page ({pull}171089[#171089]).
```

@natasha-moore-elastic looks like we'll need to refresh a few screenshots in the 8.11 ESS docs. Here are the ones that I've noticed so far -- they're both on the [Turn on the risk scoring engine](https://www.elastic.co/guide/en/security/8.11/turn-on-risk-engine.html) page:

- [`preview-risky-entities.png`](https://www.elastic.co/guide/en/security/8.11/images/preview-risky-entities.png)
- [`turn-on-risk-engine.png`](https://www.elastic.co/guide/en/security/8.11/images/turn-on-risk-engine.png)

cc: @SourinPaul @jmikell821 for visiblity

Reply:
+1 on refreshing screenshots in docs.

We should identify this `Manage/ Enablement` page uniquely or specifically. Also, we are updating the links to point to an in-product dashboard and public docs. 

So, I would suggest along the lines of `Updates references on the Entity Risk Score Management page.`

Reply:
Oh, I see. The first link (Entity Analytics dashboard) points users towards the actual EA dashboard in the Security app, and the second link (How is risk score score calculated) opens a flyout that explains the risk scoring logic and process. 

RE identifying Manage/ Enablement page uniquely or specifically:
I'm hesitant to swap "Entity Risk Score page" with "Entity Risk Score Management page" because the page appears as "Entity Risk Score" in the Security app UI. I'm also not seeing any mentions of an Entity Risk Score Management page in the [Advanced Entity Analytics](https://www.elastic.co/guide/en/security/8.11/advanced-entity-analytics-overview.html) docs, so adjusting the way we write about/reference the page introduces so inconsistency.

How about this, @SourinPaul:

Updates references on the management page for entity risk scoring. 

Another option is:

Updates the useful links provided on the Entity Risk Score page, where you can manage entity risk scoring.

Reply:
@SourinPaul quick fyi that the issue for related updates is at https://github.com/elastic/security-docs/issues/4391.